rpc/session.c

```diff
@@ -510,11 +512,9 @@ M0_INTERNAL void m0_rpc_session_establish_reply_received(struct m0_rpc_item
 	M0_ASSERT(m0_rpc_machine_is_locked(machine));
 
 	M0_ASSERT(m0_rpc_session_invariant(session));
-	M0_ASSERT_INFO(session_state(session) == M0_RPC_SESSION_ESTABLISHING,
-		       "Invalid session state: expected %s, got %s",
+	M0_LOG(M0_DEBUG, "Invalid session state: expected %s, got %s",
```

Comment:
If this happens as the description of this PR, the message would be misleading.
Can we remove the word 'Invalid' in the message string?

Reply:
Sure

Reply:
Also this can be treated as ERROR Message ?

Reply:
@vidyadhar-pinglikar @huanghua78  
If we want to make it as ERROR msg, then I'll print this only if expected and got states are not matching. what say?

Reply:
Converted DEBUG to ERROR, and logging only if session's state does not match with the expected state